README.md

```diff
@@ -73,6 +73,9 @@
    smpc_image="gpikra/coordinator:v7.0.7.4"
    db_image="mongo:5.0.8"
    queue_image="redis:alpine3.15"
+   
+   [dp]
+   enabled = false
```

Comment:
Could you also add the rest of the parameters here?

Reply:
As the feature is disabled in this deployment configuration, why should the rest of the parameters? The rest of the parameters, sensitivity and privacy budget, are irrelevant and will not be used in this kind of deployment. The same actually goes for the smpc parameters, since the enabled flag is set to "false" they should not be there.

Reply:
For the smpc it could be set to "true" and it would work. I think the same thing could happen for "dp".

Reply:
ok I added the other parameters `sensitivity` and `privacy_budget` but commented out. What I meant about the `smpc` parameters is that it is not clear if they are needed or not the way it currently is.